contracts/UpkeepRegistrationRequests.sol

```diff
@@ -319,13 +359,10 @@ contract UpkeepRegistrationRequests is Owned {
             adminAddress,
             checkData
         );
+        // approve funds
+        LINK.approve(address(keeperRegistry), amount);
```

Comment:
Note: another option here is we could approve `LINK.approve(registry, maxUint256)` on setRegistry